envs/docker-common.yml

```diff
@@ -2,30 +2,33 @@ version: '2.1'
 
 services:
   postgres:
-    image: postgres:9.6.2
+    image: postgres:15.2
```

Comment:
Getting this error when I try any `docker-compose` command from the `envs/dev` folder

`service "postgres" refers to undefined volume smart_pgdata_15_2: invalid compose project`